src/main/webapp/app/overview/course-faq/course-faq.component.ts

```diff
@@ -27,10 +29,12 @@ import { ArtemisMarkdownModule } from 'app/shared/markdown.module';
     imports: [ArtemisSharedComponentModule, ArtemisSharedModule, CourseFaqAccordionComponent, CustomExerciseCategoryBadgeComponent, SearchFilterComponent, ArtemisMarkdownModule],
 })
 export class CourseFaqComponent implements OnInit, OnDestroy {
+    faqElements = viewChildren<ElementRef>('faqElement');
     private ngUnsubscribe = new Subject<void>();
     private parentParamSubscription: Subscription;
 
     courseId: number;
+    faqId: number;
```

Comment:
Can you give this a more helping name? Why do you store only one id when you also store multiple faqs?